### ISSUE TITLE
Add feature to Display a linked row in the table that owns it.

### ISSUE THREAD

**Issue by shadyar** [dbeaver/dbeaver#19037]
**Is your feature request related to a problem? Please describe.**
In a data view, on a foreign key cell, if you open The Context Menu > Navigate > Navigate Link, The row linked shown, but there is no way to Display this row in the table that owns it.
**Describe the solution you'd like**
There should be a feature in the Context Menu and relevant shortcut to navigate to The table owns that row.
